docs/source/offloading.md

```diff
@@ -0,0 +1,45 @@
+# Background Tasks
```

Comment:
I'm not against renaming "Background Tasks" to "Offloading" but we should stick with either one or the other, not both.